build/erpnext-nginx/Dockerfile

```diff
@@ -14,7 +15,7 @@ RUN chmod +x /install_app && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
-RUN /install_app erpnext ${GIT_REPO} ${GIT_BRANCH}
+RUN /install_app erpnext ${GIT_REPO} ${GIT_BRANCH} ${FRAPPE_BRANCH}
```

Comment:
shouldn't this be

```suggestion
RUN /install_app erpnext ${GIT_REPO} ${FRAPPE_BRANCH}
```

Reply:
No. Frappe branch is additional arg now.
For regular builds behaviour doesn't change.
For PRs to have base frappe too build assets the branch need to be correct.
Branch same as fork's branch causes frappe to not get cloned during build.

Ideally the install_app script needs improvement.

Probably we should use python for the script, we'll get argparse and build better cli for install_app